README.md

```diff
@@ -1,16 +1,17 @@
 # autogluon_dashboard
 
-To view the dashboards in a local browser:
+Follow the steps below to set up autogluon-dashboard::
 1. Create a virtual environment by running: 
 ```
 python3 -m venv .venv_dash 
 source .venv_dash/bin/activate
 ``` 
 2. Install all necessary libraries by running: `pip3 install -r requirements.txt`
-3. run `python3 dashboard.py --dataset_file 'path_to_csv_file.csv' --aggreagated_file 'path_to_csv_file.csv'` and open the provided http link in your browser. 
-<br> For example, `python3 dashboard.py --dataset_file 'dev_data/all_data.csv' --aggreagated_file 'dev_data/autogluon.csv'`
+3. Install the `autogluon_dashboard` package by running: `pip3 install -e .`. You can now interact with the dashboard through the CLI command `agdash`. 
+4. To convert a Panel app to WebAssemly,  Panel provides a script that will convert the code in `app.py` into an HTML file and JS file. This can be done in one line of code as: `panel convert app.py --to pyodide-worker --out web_files/`
+The generated HTML and JS files will be found in the `web_files` folder. These can be uploaded to the desired hosting service (eg: GitHub Pages or AWS S3). 
+<br>Run `agdash --dataset_file 'path_to_csv_file.csv' --aggreagated_file 'path_to_csv_file.csv'`. 
```

Comment:
I thought the cli should do the conversion for the user? But it seems like they need to call convert then call your cli?

Reply:
No they don't need to call convert - I just gave an example on how to use panel convert. The user just has to run the agdash ... command

Reply:
If it's confusing I can remove it from the README or reword it.